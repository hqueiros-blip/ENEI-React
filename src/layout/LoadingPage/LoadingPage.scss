.loading {
  align-items: center;
  display: flex;
  flex-flow: column wrap;
  justify-content: center;

  .spinner-border {
    margin-bottom: 0.5rem;
  }
}
