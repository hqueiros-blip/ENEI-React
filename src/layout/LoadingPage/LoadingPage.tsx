import "./LoadingPage.scss";

export const LoadingPage = () => {
  return (
    <div className="loading">
      <div className="spinner-border" aria-label="Loading..." />
      <div>Loading...</div>
    </div>
  );
};
