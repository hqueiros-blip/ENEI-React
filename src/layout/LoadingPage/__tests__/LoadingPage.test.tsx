import { render } from "@testing-library/react";
import { LoadingPage } from "../LoadingPage";

describe("Loading Layout", () => {
  it("Snapshot of Loadinf", async () => {
    const { asFragment } = render(<LoadingPage />);

    expect(asFragment()).toMatchSnapshot();
  });
});
