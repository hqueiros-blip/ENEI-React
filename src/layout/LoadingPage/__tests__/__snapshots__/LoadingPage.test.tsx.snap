// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Loading Layout > Snapshot of Loadinf 1`] = `
<DocumentFragment>
  <div
    class="loading"
  >
    <div
      aria-label="Loading..."
      class="spinner-border"
    />
    <div>
      Loading...
    </div>
  </div>
</DocumentFragment>
`;
